### PR TITLE
Fixes response of agent_simulator for active-response config request

### PR DESCRIPTION
**Pull request by RamosFe**
| Issue |
|---------|
| https://github.com/wazuh/wazuh-qa/issues/1266|

# Description
Fixes the response of the `agent simulator` for the active-response configuration messages.

### CODE REVIEW

File: src/wazuh_testing/tools/simulators/agent_simulator.py [hunk 1]
```diff
@@ -577,7 +577,10 @@ def process_command(self, sender, message_list):
                     sender.send_event(self.create_event(f'#!-req {req_code} '
                                                         f'{{"error":0, "message":"ok", "data":[]}} '))
         elif command == 'getconfig':
-            response_json = '{"client":{"config-profile":"centos8","notify_time":10,"time-reconnect":60}}'
+            if "active-response" in message_list:
+                response_json = '{{"active-response":{"disabled":"no"}}'
```
Comment (Selutario) [inline src/wazuh_testing/tools/simulators/agent_simulator.py:581]:
```suggestion
                response_json = '{"active-response":{"disabled":"no"}}'
```

**Review by Selutario — CHANGES_REQUESTED** [review]
Also, add an entry in this repo's changelog please.

File: src/wazuh_testing/tools/simulators/agent_simulator.py (hunk 1 above)
Comment (RamosFe) [reply src/wazuh_testing/tools/simulators/agent_simulator.py]: Fixed in 96dd32f908fe0d1a467d002f8ad902271c1eafe5.

File: CHANGELOG.md [hunk 2]
```diff
@@ -16,6 +16,7 @@ All notable changes to this project will be documented in this file.
 ### Fixed
 - Fixed bug in the service control function for Windows agents. ([#121](https://github.com/wazuh/qa-integration-framework/pull/121))
 - Fixed bug in the RemotedSimulator udp connections mocker. ([#86](https://github.com/wazuh/qa-integration-framework/pull/86))
+- - Fixed agent_simulator response for active-response configuration commands. ([#4895](https://github.com/wazuh/wazuh-qa/pull/4895))
```
Comment (Selutario) [inline CHANGELOG.md:19]:
```suggestion
- Fixed agent_simulator response for active-response configuration commands. ([#4895](https://github.com/wazuh/wazuh-qa/pull/4895))
```

**Review by Selutario — CHANGES_REQUESTED** [review]

**Review by Selutario — APPROVED** [review]

**Pull request merged**